tests/schema/helpers.js

```diff
@@ -63,7 +63,10 @@ export function tryTest(nth, max, test, refs, schema, callback) {
   return _jsf.resolve(schema, refs).then(sample => {
     if (test.dump) {
       console.log(JSON.stringify(sample, null, 2));
-      return;
+
+      if (test.dump === 'bail') {
+        return;
```

Comment:
I thought it would be useful to dump the data without returning. Adding this logic to retain the original behavior if needed.